srcpkgs/aquatone/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'aquatone'
+pkgname=aquatone
+version=1.4.3
+revision=1
+build_style=go
+hostmakedepends="git"
+go_import_path=github.com/michenriksen/aquatone
```

Comment:
go_import_path above hostmakdepends below build_style

Reply:
Fixed that. I was pasting a line too low using `dd` `p` in vim.